server/src/main/java/org/opensearch/search/profile/query/ConcurrentQueryProfileBreakdown.java

```diff
@@ -23,7 +27,22 @@
  * @opensearch.internal
  */
 public final class ConcurrentQueryProfileBreakdown extends ContextualProfileBreakdown<QueryTimingType> {
+    public static final String MAX_END_TIME_SUFFIX = "_max_end_time";
+    public static final String MIN_START_TIME_SUFFIX = "_min_start_time";
+    private static final String AT = "@";
+    private static final String LATEST_LEAF_END_TIME = "latest_leaf_end_time";
+    private static final String MAX_PREFIX = "max_";
+    private static final String MIN_PREFIX = "min_";
+    private static final String AVG_PREFIX = "avg_";
+    private static long breakdownMaxEndTime = 0;
+    private static long breakdownMinStartTime = Long.MAX_VALUE;
+    private static long maxSliceNodeTime = Long.MIN_VALUE;
+    private static long minSliceNodeTime = Long.MAX_VALUE;
+    private static long avgSliceNodeTime = 0L;
+    private static long nodeTimeSum = 0L;
     private final Map<Object, AbstractProfileBreakdown<QueryTimingType>> contexts = new ConcurrentHashMap<>();
+    private List<String> collectorList = new ArrayList<>();
+    private Map<LeafReaderContext, String> leafToCollector = new HashMap<>();
```

Comment:
I don't think we need these 2 and instead just keep track of `Map<Collector, List<LeafReaderContext>>` which will give the slice to leaves mapping.

Reply:
Hi @sohami, thank you for the review. We can also achieve that using your method.